lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -1825,21 +1769,10 @@ void PatchEntryPointMutate::addUserDataArgs(SmallVectorImpl<UserDataArg> &userDa
       if (!isComputeWithCalls() && (!descSetUsage || descSetUsage->users.empty()))
         break;
 
-      unsigned userDataValue = node.offsetInDwords;
-      if (m_pipelineState->getShaderOptions(m_shaderStage).updateDescInElf && m_shaderStage == ShaderStageFragment) {
-        // Put set number to register first, will update offset after merge ELFs
-        // For partial pipeline compile, only fragment shader needs to adjust offset of root descriptor.
-        // This is part of the original "partial pipeline compile" scheme, and it uses a magic number for the
-        // PAL metadata register value because the code to fix it up in llpcElfWriter.cpp just fixes up any
-        // register with the magic value, and hopes it lucks out by not getting a false positive.
-        // TODO: Remove all that code once the new "shader/part-pipeline compile" scheme can replace it.
-        static const unsigned DescRelocMagic = 0xA5A5A500;
-        userDataValue = DescRelocMagic | node.innerTable[0].set;
-      }
```

Comment:
Should the whole `DescRelocMagic` thing be removed now, as it seems this was the only place emitting it?

See https://github.com/search?q=repo%3AGPUOpen-Drivers%2Fllpc+descrelocmagic&type=code

Reply:
Yes, makes sense. I will add that.